applications/GeoMechanicsApplication/custom_elements/geo_linear_truss_element.cpp

```diff
@@ -129,8 +129,8 @@ void GeoLinearTrussElement<TDim,TNumNodes>::
         ConstitutiveLaw::Parameters Values(this->GetGeometry(),
                                            this->GetProperties(),
                                            rCurrentProcessInfo);
-        Vector temp_strain = ZeroVector(1);
```

Comment:
stress and strains have always size 1.

Reply:
mStressVectorSize is set to 1 in  the .h file, I changed "1" to "mStressVectorSize" in order to stay consistent with "mInternalStressesFinalizedPrevious" as defined in "ResetConstitutiveLaw()". Like this, memory issues cannot occur when adding "temp_stress" and "mInternalStressesFinalizedPrevious"